email.rst

```diff
@@ -118,6 +118,9 @@ The ``$message`` object supports many more options, such as including attachment
 adding HTML content, and much more. Refer to the `Creating Messages`_ section
 of the Swift Mailer documentation for more details.
 
+You can also use :doc:`Monolog </logging>` and configure it to
```

Comment:
It feels like it could be a nice addition in this page, but does not really fit anywhere. What about a new section after `Troubleshooting` so an anchor could be visible at the top of the page, then just adding the link in this new last section of the page as done currently?